fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rewrite/ScalarOperatorFunctions.java

```diff
@@ -1278,4 +1284,17 @@ public static ConstantOperator coalesce(ConstantOperator... values) {
         }
         return values[values.length - 1];
     }
+
+    @ConstantFunction(name = "url_extract_parameter", argTypes = {VARCHAR, VARCHAR}, returnType = VARCHAR)
+    public static ConstantOperator urlExtractParameter(ConstantOperator url, ConstantOperator parameter)
+            throws URISyntaxException {
```

Comment:
Could we catch the URISyntaxException and return a null value ?

Reply:
+1

Reply:
done